x-pack/legacy/plugins/apm/public/components/shared/SelectWithPlaceholder/index.tsx

```diff
@@ -20,7 +20,7 @@ export const SelectWithPlaceholder: typeof EuiSelect = props => (
     {...props}
     options={[
       { text: props.placeholder, value: NO_SELECTION },
-      ...props.options
+      ...props.options!
```

Comment:
We don't have a type guarantee here, what do you think about `...(props.options || [])`?

Reply:
Pushed your suggestion, thanks for catching